api/geth_backend.go

```diff
@@ -350,8 +350,8 @@ func (b *GethStatusBackend) mergeConfig(n *params.NodeConfig) (*params.NodeConfi
 		return nil, err
 	}
 
-	// Overwrite db configuration (only adds new values)
-	if err := mergo.Merge(conf, n); err != nil {
+	// Overwrite db configuration
+	if err := mergo.MergeWithOverwrite(conf, n); err != nil {
```

Comment:
MergeWithOverwrite is deprecated
```suggestion
	if err := mergo.Merge(conf, n, mergo.WithOverride); err != nil {
```